library-backend/library-app/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
This file should live in a `docker` folder, right next to the `java` folder. Same for the compose.